### PR TITLE
Adds a toxins and burn medkit to the mining outpost and adds in a xenomorph toy!

### PR DESCRIPTION
**Pull request by Ultimate-Chimera**
I decided to add a toxins and burn kit to the medbay in the outpost because it lacked one... Why?

I also added in a xenomorph toy from /tg/ https://github.com/tgstation/-tg-station/pull/11411

ss 2015-09-07 at 07 20 39

### CODE REVIEW

**Comment by Spacemanspark** [comment]
:+1:

**Comment by Fox-McCloud** [comment]
Don't think this was mapmerged? Hard to tell.

There shouldn't be *two* booze dispensers and vendors though.

Also, you'll need fire alarms in there as well.

**Comment by Ultimate-Chimera** [comment]
Alright, I just had no idea what to put in there but I will remove the two extras and add a fire alarm.

**Comment by Markolie** [comment]
Mapmerge doesn't work at the moment anyway.

**Comment by Ultimate-Chimera** [comment]
Also what do you mean by mapmerge?

**Comment by Markolie** [comment]
http://www.ss13.eu/wiki/index.php/Map_Merger

**Comment by Ultimate-Chimera** [comment]
Oh... Alright, that made little sense to me but thanks for the information.

**Comment by Ultimate-Chimera** [comment]
Commits are so annoying, but I removed the second booze-o-mat and dispense and added a fire alarm and fire lock under the airlock.

**Comment by Ultimate-Chimera** [comment]
Do you know how to fix the commits? I tried git rebase -i and everything.

**Comment by Fox-McCloud** [comment]
This is nitpicking, but ideally the dispensers should be next to one another so you're not having to walk all over to make a singular drink.

**Comment by Ultimate-Chimera** [comment]
You can stand in the middle? But I can change that.

**Comment by Ultimate-Chimera** [comment]
Fixed.

**Comment by SpaceTimeNow** [comment]
If we sart nitpicking, then we could also add a holopad :)
Other than that :+1:

**Comment by M3hillus** [comment]
:-1: Adding a full on bar to the mining outpost just serves to eliminate even more interaction with the station crew proper.

**Comment by Alex6511** [comment]
I agree with @M3hillus :-1:

**Comment by Ultimate-Chimera** [comment]
God damn it, I want to update the asteroid badly, tell me what you want then.

**Comment by M3hillus** [comment]
Well, I'm currently working on the main area, we could totally collaborate and swap ideas and stuff on that, as that area needs a bit more love than the hab unit, TBH.

**Comment by Ultimate-Chimera** [comment]
If you don't want a bar, I can just do the medkit thing for the outpost.

**Comment by Keilden** [comment]
Add a trophy room that miners can drag dead things into.

**Comment by Taaketa** [comment]
Since it's being discussed here. I'd suggest the following:

Medbay. Make biggger by one tile wide and off set the door by one tile right from it's current position.

Add a Burn med-kit to medbay.

Keep the hot drinks, cigerette and food dispensor machines, espiecally keep the donk pockets and microwave. I use them for staying alive on the asteroid.

Change the disposal pipes. Make them rapid transit to the North, West Mining outposts. Make two entrances one that that goes west and one that goes north. Currently both pipes feed into a single exit that could be changed.

Make the furnance area smaller and simpler. Remove the conveyers, place the furnance with an "entry point" similar to the setup in cargo office. Then have the stacking machine below it to retrieve stacks of material.

**Comment by Ultimate-Chimera** [comment]
There is a burn kit, there's everything BUT a toxins kit.

**Comment by Taaketa** [comment]
Last I checked there was only Brute, First-Aid and Oxygen depreviation.

**Comment by Ultimate-Chimera** [comment]
Nope, all except toxins.. Which is needed.

File: code/game/objects/items/toys.dm [hunk 1]
```diff
@@ -1318,3 +1319,32 @@ obj/item/toy/cards/deck/syndicate/black
...
+
+/obj/item/toy/toy_xeno/attack_self(mob/user)
+	if(cooldown <= world.time)
+		cooldown = (world.time + 50) //5 second cooldown
+		user.visible_message("<span class='notice'>[user] pulls back the string on [src].</span>")
+		icon_state = "[initial(icon_state)]_used"
+		sleep(5)
+		audible_message("<span class='danger'>\icon[src] Hiss!</span>")
+		var/list/possible_sounds = list('sound/voice/hiss1.ogg', 'sound/voice/hiss2.ogg', 'sound/voice/hiss3.ogg', 'sound/voice/hiss4.ogg')
+		var/chosen_sound = pick(possible_sounds)
```
Comment (tigercat2000) [inline code/game/objects/items/toys.dm]: This var is redundant, the playsound should just use pick(possible_sounds)

**Comment by Taaketa** [comment]
Really, nope. Currently on live there is no burn kit.

http://i.imgur.com/4EpqcIh.png

Screen shot is from round start. Note the floor tab on the rigt hand side.

**Comment by Ultimate-Chimera** [comment]
I can fix that. @tigercat2000

**Comment by tigercat2000** [comment]
@Ultimate-Chimera Just replace playsound(get_turf(src), chosen_sound, 50, 1) with playsound(get_turf(src), pick(possible_sounds), 50, 1)

**Comment by Ultimate-Chimera** [comment]
Alright, thanks tiger!

**Comment by Ultimate-Chimera** [comment]
Fixed.

**Comment by tigercat2000** [comment]
You should also delete the chosen_sound var, or the compile will throw warnings.

**Comment by Ultimate-Chimera** [comment]
I think I did, lemme check.

**Comment by Ultimate-Chimera** [comment]
Nope, do I just delete the line or what?

**Comment by Ultimate-Chimera** [comment]
var/chosen_sound = pick(possible_sounds)

**Comment by tigercat2000** [comment]
Yeah, that line can be deleted entirely.

**Comment by Ultimate-Chimera** [comment]
Done.

**Comment by Ultimate-Chimera** [comment]
Do you support this, tiger? :)

**Comment by tigercat2000** [comment]
I don't really have an opinion on it.

**Comment by Ultimate-Chimera** [comment]
Oh, alright... I really want this merged in because of the medkits and the cool toy! :)

**Comment by Fox-McCloud** [comment]
Merge conflicts.

I also tend to agree that having a bar is probably a bit too much.

**Comment by Ultimate-Chimera** [comment]
Alright, I'll remove the bar and just have the much needed kits.

**Comment by Ultimate-Chimera** [comment]
And update the map, of course! :)

**Comment by Ultimate-Chimera** [comment]
Wait, how do I fix the conflicts? I put the recent map update in from the other PR and just put the two kits in that were gone... And it still has conflicts?

**Comment by Ultimate-Chimera** [comment]
Fox, help me...

**Comment by M3hillus** [comment]
I don't know if this is the "Right" way, but what I had to to do was go to my branch on github, set it as the head branch, and merge the paradise master to it, and re-do the changes.

**Comment by Ultimate-Chimera** [comment]
Why didn't you wait for my PR to go through? You screwed me up! :(

**Comment by Spacemanspark** [comment]
RIP in pepperoni.

**Comment by Ultimate-Chimera** [comment]
Can somebody with github experience help me out here?

**Comment by Fox-McCloud** [comment]
Generally speaking, (for me anyway), it's easier to redo map changes than to resolve the conflicts with them.

**Comment by Ultimate-Chimera** [comment]
And how would I do that? After I do that I can make changes, right?

**Comment by Ultimate-Chimera** [comment]
Fox?

**Comment by tigercat2000** [comment]
these questions aren't answerable without knowing a few things
namely
1. what git program are you using
2. do you have everything setup correctly

**Comment by Ultimate-Chimera** [comment]
I'm using the official github app.

**Comment by tigercat2000** [comment]
I have no experience with the github app, so consider my advice out.

**Comment by Ultimate-Chimera** [comment]
Oh... What about git shell?

**Comment by tigercat2000** [comment]
In git shell, the command to update would be `git fetch upstream`, followed by `git merge upstream/master`. If merge conflicts are present, it will require you to resolve the merge conflicts and manually complete the update via `git add` and `git commit`.

**Comment by Ultimate-Chimera** [comment]
It says this fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists. when trying to fetch upstream

**Comment by tigercat2000** [comment]
It's possible that the github app hasn't setup the remote correctly. In that case, you would need to run `git remote add upstream https://github.com/ParadiseSS13/Paradise.git` before trying the other commands.

**Comment by Ultimate-Chimera** [comment]
That fixed it, thanks.

**Comment by Ultimate-Chimera** [comment]
Ah, you know how to uh... Make this work? It says this when I type this: Paradise\_maps\map_files\cyberiad\z5.dmm fatal: pathspec 'Paradise/_maps/map_files/cyberiad/z5.dmm' did not match any files

**Comment by Ultimate-Chimera** [comment]
I mean I type git add Paradise/_maps/map_files/cyberiad/z5.dmm

**Comment by tigercat2000** [comment]
um, guessing you have git shell open in the directory
in which case it would be _maps/map_files/cyberiad/z5.dmm

if you want to nuke the changes and just make the map equal to upstream, run `git checkout upstream/master _maps/map_files/cyberiad/z5.dm`

**Comment by Ultimate-Chimera** [comment]
error: pathspec '_maps/map_files/cyberiad/z5.dm' did not match any file(s) known to git.

**Comment by tigercat2000** [comment]
what directory is git shell in? it should say somewhere.

**Comment by Ultimate-Chimera** [comment]
Just the default directory... I selected Paradise though.

**Comment by Ultimate-Chimera** [comment]
I did manage to get git add to work, but... Git commit.. U       _maps/map_files/cyberiad/z5.dmm
U       icons/obj/toy.dmi
error: 'commit' is not possible because you have unmerged files.
hint: Fix them up in the work tree,
hint: and then use 'git add/rm <file>' as
hint: appropriate to mark resolution and make a commit,
hint: or use 'git commit -a'.
fatal: Exiting because of an unresolved conflict.

**Comment by Ultimate-Chimera** [comment]
I hate github, it is so confusing... :(

**Comment by Ultimate-Chimera** [comment]
Holy crap, I somehow got 243 changes.

**Comment by Ultimate-Chimera** [comment]
Oh crap... This MIGHT be a problem.

**Comment by Ultimate-Chimera** [comment]
I checked out all the commits, but I didn't push... Help!

**Comment by tigercat2000** [comment]
you need to run `git push origin master`

**Comment by Ultimate-Chimera** [comment]
The map is the same, I do not know how to fix all these changes now without re-forking and re-cloning to desktop... Why can't it be merged? :(

**Comment by Ultimate-Chimera** [comment]
This guy screwed me up 100% because he got his PR merged first.

**Comment by Ultimate-Chimera** [comment]
git push origin master
To https://github.com/Ultimate-Chimera/Paradise.git
 ! [rejected]        master -> master (non-fast-forward)
error: failed to push some refs to 'https://github.com/Ultimate-Chimera/Paradise.git'
hint: Updates were rejected because a pushed branch tip is behind its remote
hint: counterpart. Check out this branch and integrate the remote changes
hint: (e.g. 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.

**Comment by Ultimate-Chimera** [comment]
This is a problem. git commit -a
On branch ParadiseSS13/master
Your branch is ahead of 'origin/master' by 107 commits.
  (use "git push" to publish your local commits)

nothing to commit, working directory clean

**Comment by Ultimate-Chimera** [comment]
I think I should just re-do the entire PR, like re-fork and everything, because I do NOT know how to fix all this crap I did because of this guy that screwed my PR up. :(

**Comment by Ultimate-Chimera** [comment]
Unless if you can help me, tigercat.

**Comment by Ultimate-Chimera** [comment]
Screw it, just gonna reinstall everything.

**Comment by Ultimate-Chimera** [comment]
I hate when my PR gets closed too because my commits remain from the last one so I need to reinstall the whole thing again.

**Pull request closed without merging**